### PR TITLE
Add predawn theme

### PR DESCRIPTION
**Pull request by jaard**
added matching theme for predawn-ui and predawn-syntax

### CODE REVIEW

**Comment by ivankravets** [comment]
Did you test that?

**Comment by jaard** [comment]
Yes, I copied the colors from the predawn package and it looks fine.
I'm running Atom 1.18.0 on OSX, haven't tried it on another machine.

File: lib/platformio-ide-terminal.coffee [hunk 1]
```diff
@@ -147,6 +147,7 @@ module.exports =
             'dracula',
             'one-dark',
             'christmas'
+            'predawn'
```
Comment (ivankravets) [inline lib/platformio-ide-terminal.coffee]: Missed comma?

Comment (jaard) [reply lib/platformio-ide-terminal.coffee]: True, sorry about that. Changed it.

**Pull request merged**